examples/physical_properties/chargeability/ColeCole.ipynb

```diff
@@ -0,0 +1,269 @@
+{
+ "cells": [
```

Comment:
can we put this in the EM examples repository? https://github.com/ubcgif/em_examples